packages/monorepo-builder/packages/Release/Process/ProcessRunner.php

```diff
@@ -8,7 +8,7 @@
 use Symfony\Component\Process\Exception\ProcessFailedException;
 use Symfony\Component\Process\Process;
 
-final class ProcessRunner
+final class ProcessRunner implements ProcessRunnerInterface
```

Comment:
This interface is not needed

Reply:
I've added this interface to make it possible to mock this service. This topic is connected with the one from the comment above.